etcdctl/ctlv3/command/auth_command.go

```diff
@@ -40,6 +41,14 @@ func newAuthEnableCommand() *cobra.Command {
 	}
 }
 
+func newAuthenticateCommand() *cobra.Command {
```

Comment:
yea... let's keep this simple. let's add this back when we are sure we need this.